src/style.css

```
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf),
+    url(/fonts/Avenir-Heavy.ttf),
+    url(/fonts/Avenir.ttc);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  padding-left: 8px;
+  padding-top: 20px;
+  padding-right: 8px;
+}
+
+.form__search--big {
+  padding-top: 23px;
+  padding-bottom: 23px;
+}
+
+.form__search--big:hover {
+  box-shadow: 0 4px 4px 0 rgba(0,0,0, 0.25);
+}
+
+.form__input--big::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.form__search--small {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.form__search--small:focus {
+  font-family: "Avenir", sans-serif;
```

```suggestion
  font-family: inherit;
```
it's better to use inherit font family